jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/Content.java

```diff
@@ -155,6 +155,22 @@ public static void copy(Source source, Sink sink, Chunk.Processor chunkProcessor
      */
     public interface Source
     {
+        /**
+         * Factory of {@link Content.Source}.
+         */
+        interface Factory
```

Comment:
Can you also use this interface for `MultiPart` and any other places that are appropriate?

Reply:
Done, let's discuss it where the modifications were made.